src/TreeSitter/Language.hs

```diff
@@ -1,54 +1,45 @@
 {-# LANGUAGE TemplateHaskell #-}
-module TreeSitter.Language where
-
-import Data.Char
-import Data.Function ((&))
-import Data.Ix (Ix)
-import Data.Traversable (for)
-import Data.List.Split
-import Data.Word
-import Foreign.C.String
-import Foreign.Ptr
-import Language.Haskell.TH
-import Language.Haskell.TH.Syntax
-import System.Directory
-import System.FilePath.Posix
+module TreeSitter.Language
+( module TreeSitter.Language
+, module TreeSitter.Symbol
```

Comment:
I’m re-exporting `TreeSitter.Symbol` for backwards compatibility; didn’t want to break @aymannadeem’s in-flight work in #145.